pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -641,6 +649,33 @@ def U(self, sto, T, lithiation=None):
             out.print_name = r"U_\mathrm{p}(c^\mathrm{surf}_\mathrm{s,p}, T)"
         return out
 
+    def hysteresis(self, sto):
+        """Dimensional hysteresis [V]"""
+        Domain = self.domain.capitalize()
+        # tol = pybamm.settings.tolerances["U__c_s"]
+        # sto = pybamm.maximum(pybamm.minimum(sto,1-tol),tol)
+        inputs = {f"{self.phase_prefactor}{Domain} particle stoichiometry": sto}
+        lith_ref = pybamm.FunctionParameter(
+            f"{self.phase_prefactor}{Domain} electrode lithiation OCP [V]", inputs
+        )
+        delith_ref = pybamm.FunctionParameter(
+            f"{self.phase_prefactor}{Domain} electrode delithiation OCP [V]", inputs
+        )
+        h_ref = abs(delith_ref - lith_ref) / 2
+        # h_ref = pybamm.FunctionParameter(
+        #     f"{self.phase_prefactor}{Domain} electrode OCP hysteresis [V]", inputs
+        # )
+        return h_ref
+
+    def Q(self, sto):
```

Comment:
What if the active material volume fraction changes? Maybe this is fine as the definition of capacity within the context of the hysteresis model, but again that might be an argument for it to be defined locally rather than here.

Reply:
If I am not mistaken, when the active material volume fraction changes, it is captured in this definition as the `epsilon_s_av` parameter. Do you think it's worth trying to put this in the hysteresis submodel class? If so, which method would it be defined in?

Reply:
The amount of active material is specified by the `active_material` submodels. Constant (default) active material takes the value of that parameter, but if loss of active material is enabled, that parameter is only the initial condition

Reply:
Understood! I will take a closer look at that.

Reply:
Done. I changed the calculation to use the epsilon variable instead of parameter.